pontoon/base/models.py

```diff
@@ -622,7 +622,7 @@ def get_chart_dict(cls, obj):
                 'approved_strings': obj.approved_strings,
                 'translated_strings': obj.translated_strings,
                 'fuzzy_strings': obj.fuzzy_strings,
-                'approved_share': round(obj.approved_strings / obj.total_strings * 100),
+                'approved_share': int(math.floor(obj.approved_strings / obj.total_strings * 100)),
```

Comment:
Hi Victor, it's nice to see your patch. After a fast look: I'm pretty sure that math.floor isn't equivalent to the round in this context. I would suggest you to just cast the results of round() to integer.